scripts/ci/run-ci-tests.sh

```diff
@@ -293,7 +293,14 @@ ip net add test
 # This is a temporary workaround until fixed in the kernel.
 # The kernel currently does not show correct device and inode numbers in /proc/pid/maps
 # for stackable file systems.
-if capsh --supports=cap_checkpoint_restore && unshare -c /bin/true && [ ! -e /run/.containerenv ]; then
+skip=0
```

Comment:
@avagin I think we need to update the comment above as https://github.com/torvalds/linux/commit/3efdc78fdc21 has been merged.